packages/collector/test/tracing/logger/pino/test.js

```diff
@@ -10,8 +10,8 @@ const supportedVersion = require('@instana/core').tracing.supportedVersion;
 const constants = require('@instana/core').tracing.constants;
 const config = require('../../../../../core/test/config');
 const testUtils = require('../../../../../core/test/test_util');
-const ProcessControls = require('../../../test_util/ProcessControls');
 const globalAgent = require('../../../globalAgent');
+const appControls = require('./controls');
```

Comment:
issue: Please do not touch this test. Add a new internal logging integration test.